From 1617b20950624b199d3aaceff7e1f2967e558950 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Petr=20P=C3=ADsa=C5=99?= <ppisar@redhat.com>
Date: Mon, 17 Jan 2022 14:57:40 +0100
Subject: [PATCH] Fix test plan
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The test plan was wrong:

$ prove test.pl
test.pl .. All 2 subtests passed

Test Summary Report
-------------------
test.pl (Wstat: 0 Tests: 3 Failed: 1)
  Failed test:  3
  Parse errors: Bad plan.  You planned 2 tests but ran 3.
Files=1, Tests=3,  0 wallclock secs ( 0.01 usr +  0.02 sys =  0.03 CPU)
Result: FAIL

This patch updates it to include the third test.

Signed-off-by: Petr Písař <ppisar@redhat.com>
---
 test.pl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test.pl b/test.pl
index c25b2cb..94f776c 100644
--- a/test.pl
+++ b/test.pl
@@ -6,7 +6,7 @@
 # Change 1..1 below to 1..last_test_to_print .
 # (It may become useful if the test is moved to ./t subdirectory.)
 
-BEGIN { $| = 1; print "1..2\n"; }
+BEGIN { $| = 1; print "1..3\n"; }
 END {print "not ok 1\n" unless $loaded;}
 require 5.006;
 use Config qw(%Config);
-- 
2.34.1

